### ISSUE TITLE
Slider and the aria-readonly attribute

### ISSUE THREAD

**Issue by curtbellew** [w3c/aria#1722]
Currently aria-readonly is a supported property for the slider in ARIA.  The HTML spec for the range element does not accept the readony property.  Browsers don't support the readonly attribute on the ARIA slider and I suspect that's a result of the HTML spec specifically saying readonly 'must not be specified' on range.  In this case ARIA is correct in my view in that slider should have a readonly property.  Otherwise, in cases where we need to present the value of the slider without allowing an update to it, we need to use some other role to represent it and that's confusing.  If something is a slider then it should be a slider whether I can update it or not.  The disabled attribute is supported but disabled means that the element is of no use - disabled elements are effectively a decorative element left on the page so as not to cause layout issues.

**Comment by jnurthen** [comment]
@curtbellew this sounds like there are browser bugs. Have you filed them?

**Comment by JAWS-test** [comment]
When I use `aria-readonly=true` at `<input type=range>` or `role=slider`, browsers pass both the ARIA attribute aria-readonly and the status that the value cannot be changed to the operating system's Accessibility API (e.g. from Chrome to Windows' UIA). However, NVDA and JAWS ignore this information in the API, so it seems to be a screen reader bug.

**Comment by JAWS-test** [comment]
Have JAWS filed a bug (https://github.com/FreedomScientific/VFO-standards-support/issues/620). Someone else can do the same for NVDA
